api/python/cellxgene_census/tests/experimental/pp/test_online.py

```diff
@@ -54,14 +59,38 @@ def test_meanvar(matrix: sparse.coo_matrix, n_batches: int, stride: int, ddof: i
     assert batches_var.shape == (n_batches, matrix.shape[1])
 
     dense = matrix.toarray()
-    assert allclose(all_u, dense.mean(axis=0))
-    assert allclose(all_var, dense.var(axis=0, ddof=ddof, dtype=np.float64))
 
-    matrix = matrix.tocsr()  # COO not conveniently indexable
-    for batch in range(n_batches):
-        dense = matrix[batches == batch, :].toarray()
-        assert allclose(batches_u[batch], dense.mean(axis=0))
-        assert allclose(batches_var[batch], dense.var(axis=0, ddof=ddof, dtype=np.float64))
+    if should_exclude_zeros:
+        mask = np.ones(matrix.shape)
+        r, c = matrix.tolil().nonzero()
+        for x, y in zip(r, c):
+            mask[x, y] = 0
+        masked = ma.masked_array(dense, mask=mask)  # type: ignore[no-untyped-call, var-annotated]
```

Comment:
Please don't disable mypy errors when they can just be fixed with a type declaration.

E.g., you can remove the `var-annotated` with a type declaration:
>         masked: np.ma.MaskedArray[np.bool_, np.dtype[np.float64]] = ma.masked_array(dense, mask=mask)  # type: ignore[no-untyped-call]